vue/components/ui/templates/error-part/error-part.vue

```diff
@@ -1,12 +1,19 @@
 <template>
     <div class="error-part">
         <button-icon
-            class="button-icon-back"

```

Comment:
This is a breaking change and could be problematic because if automated tests depend on it, so keep it. You can add a button-icon class though.